app/views/catalog/_fixity_dashboard_home.html.erb

```diff
@@ -1,12 +1,14 @@
-<div class="row">
-  <div class="col-md-12">
-    <div class="panel panel-fixity panel-classify-work">
-      <div class="panel-heading">Fixity</div>
-      <div class="panel-body">
-        <ul>
-          <li><%= link_to 'Fixity Dashboard', fixity_dashboard_path %></li>
-        </ul>
+<% if current_or_guest_user.admin? %>
```

Comment:
Can you just do `<% if can?(:read, :fixity) %>`?

Reply:
Also can we get a spec for viewing the link?

Reply:
oh yeah totally. I put that in the controller after restricting the view. will change it to use `can?`

Reply:
spec added. ready for re-review.